util/helpers.py

```diff
@@ -1,8 +1,147 @@
+import tensorflow as tf
+import tensorflow.compat.v1 as tfv1
+from util.flags import create_flags, FLAGS
+from util.logging import log_info, log_error, log_debug, log_progress, create_progressbar
+import sys
+
+def load_model(session, checkpoint_filename, drop_source_layers, use_cudnn):
+    r'''
+    if use_cudnn:
+        move all Adam vars to init_vars
+    if drop_souce_layers>0:
+        move all dropped layers to init_vars
+    '''
```

Comment:
Either remove this docstring or improve it.

Reply:
removed

Reply:
outdated